src/applications/personalization/profile/components/Profile.jsx

```diff
@@ -294,20 +294,25 @@ const mapStateToProps = state => {
   const showLoader =
     !hasLoadedAllData || (!isLOA3 && !isLOA1 && currentlyLoggedIn);
 
+  const shouldShowDirectDeposit = () => {
```

Comment:
I think this function is a lot easier to read compared to the single long expression using a bunch of logical and and logical or operators. And the logic has been simplified as well.